lib/gradle.properties

```diff
@@ -0,0 +1,23 @@
+VERSION_NAME=1.0.0
+
+GROUP=com.freeletics.statelayout
+POM_ARTIFACT_ID=statelayout
+POM_NAME=StateLayout
+POM_PACKAGING=aar
+
+POM_DESCRIPTION=StateLayout
+POM_INCEPTION_YEAR=2020
+
+POM_URL=http://github.com/freeletics/StateLayout/
+POM_SCM_URL=http://github.com/freeletics/StateLayout/
+POM_SCM_CONNECTION=scm:git:git://github.com/freeletics/StateLayout.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/freeletics/StateLayout.git
+
+POM_LICENCE_NAME=The Apache Software License, Version 2.0
+POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=Freeletics
+POM_DEVELOPER_NAME=Freeletics
```

Comment:
You could add a `POM_DEVELOPER_URL` that links to `https://freeletics.com` or `https://freeletics.engineering/`